src/main/java/seedu/address/logic/commands/AddLessonCommand.java

```diff
@@ -17,6 +18,8 @@
 public class AddLessonCommand extends Command {
 
     public static final String COMMAND_WORD = "addc";
+    public static final String OVERLOAD_IDENTICAL_TIMING =
+            "There are already " + EdulogCalendar.MAX_IDENTICAL_TIMING + " lessons with the same start and end time.";
```

Comment:
Do you think it would be better to check for simultaneous timings instead? This is to make sure that the calendar doesnt have too many things at the same timing. For example, we wouldnt allow 'Monday 1200 1900', 'Monday 1600 2000', 'Sunday 2300 1930' since they all share the same time 1600 to 1900.

Reply:
good call, updated, and also updated tests accordingly